prime-router/docs/onboarding-users/sender-onboarding/sftp-ingestion.md

```diff
@@ -0,0 +1,42 @@
+# SFTP Ingestion
+
+Reports can be sent into via ReportStream via SFTP in addition to the REST endpoint.
+This functionality is enabled by configuring an Azure storage account with SFTP enabled and an azure function 
+(submitSFTP) with a BlobTrigger that consumes reports as they are added.  The sender is then parsed from the filepath.
+
+## Setting up a sender
+
+These steps apply to an already existing azure storage container with SFTP enabled.
+
+1. Create the sender in the settings.
+1. Go to the storage container -> containers and create the directory for the sender in the `reports` container
+    - The directory structure needs to align with the organization and sender name.  For example, for the `ignore.ignore-full-elr`
+   the directory structure should be `reports/ignore/ignore-full-elr`
+    - **Note: because this is a directory make sure that the organization and sender name only contains valid characters for a directory**
+![](SFTP-directories.png)
+1. Go to the storage container -> SFTP
+![](SFTP-settings.png)
+1. Create the user for the sender. The username can only contain lower case letters so the recommendation is to combine
+the organization and sender. For example: ignoreignorefullelr.  Then check password
+1. Check password
+![](SFTP-add-user.png)
+1. Go to permissions and grant access to the reports container with the `write` permission and then specify the home 
```

Comment:
Does the sender have write access to the entire sftp container or just their own directory?

Reply:
So long term, we'll need devops to manage this, but we'll be able to write fine grained ACLs to manage this.

In the short term, we'll only have the one sender who will only have write access and be defaulted to the right spot.  Since they won't have list it will be hard to get somewhere else without being malicious

Reply:
I'm good with it as long as we put a devops ticket in to keep track of how we want our permissions to ultimately work.

Reply:
👍🏻 I'll add another ticket here as well

Reply:
https://github.com/CDCgov/prime-reportstream/issues/16261